### PR TITLE
Feature/animations

### LINKED ISSUE #31: Create a StudentDashboard Component
It must: 
 - [x] Have a Question
 - [x] Have possible Answers
 - [x] Render the students Name
 - [x] Render a Reading

### LINKED ISSUE #33: Find Dyslexia friendly fonts
- [ ] Find dyslexia-friendly fonts

### LINKED ISSUE #37: Add animations to student dash
- [ ] Add animations to student dash

### PR DESCRIPTION
**Pull request by Billwilke42**
## Pull request checklist
Please check if your PR fulfills the following requirements:
- [x] Lint (`npm run lint`) has passed locally and any fixes were made for failures
## Pull request type
 
 
Please check the type of change your PR introduces:
- [x] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Testing
- [ ] Other (please describe): 
## What is the current behavior?

- No animations, netlify deploy is not working correctly, scores are only single integers not percent, non dyslexic friendly fonts
## What is the new behavior?

- adds animation and motivational talk, scores are percent, and hopefully fixes netlify deploy, adds dyslexic friendly font
## Does this introduce a breaking change?
- [ ] Yes
- [ ] No

## Other information

run 'npm install react-responsive-spritesheet --save' to install the spritesheet webpack.

<<<<<<< HEAD

>>>>>>> 49ea6a2e4d81cd8aaf5ad430b688adb405c87982

closes #33 
closes #31 
closes #37

### CODE REVIEW

File: src/StudentDashboard/MoodForm.js [hunk 1]
```diff
@@ -9,10 +9,14 @@ import { connect } from 'react-redux'
...
-  //eslint-disable-next-line
+
   const [studentMoodInput, setStudentMoodInput] = useState('')
+  
   let [isSubmitted, setIsSumbitted] = useState(false)
-
+  
+  const score = (props.score / props.lesson.questions.length)
+  
+  const roundedScore = Math.round(score*100)
```
Comment (Melizzo) [inline src/StudentDashboard/MoodForm.js:20]: Nice - this great data that the BE will really able to use!!

File: src/StudentDashboard/MoodForm.scss [hunk 2]
```diff
@@ -11,7 +11,8 @@
   border: none;
   border-radius: 5px;
   cursor: pointer;
-  border: 4px ridge #0072B2
+  border: 4px ridge #0072B2;
+  font-family: 'OpenDyslexic-regular';
 }
```
Comment (Melizzo) [inline src/StudentDashboard/MoodForm.scss:16]: YES!!! This is awesome.  Accessibility here we come!

File: src/StudentDashboard/StudentDashboard.js [hunk 3]
```diff
@@ -8,6 +8,8 @@ import { getLesson } from '../thunks/getLesson'
 import { bindActionCreators } from 'redux'
 import QuestionArea from './QuestionArea';
 import { connect } from 'react-redux'
+import Spritesheet from 'react-responsive-spritesheet'
+import panda from './assets/PandaJumpAndRandom.png'
```
Comment (Melizzo) [inline src/StudentDashboard/StudentDashboard.js:13]: 🐼 !!!!

File: package.json [hunk 4]
```diff
@@ -11,6 +11,7 @@
     "react-browser-router": "^2.1.2",
     "react-dom": "^16.13.1",
     "react-redux": "^7.2.1",
+    "react-responsive-spritesheet": "^2.3.9",
```
Comment (KyleWong2510) [inline package.json:14]: Say whaaa?  Responsive?  Awesome.

File: src/TeacherHeader/TeacherHeader.js [hunk 5]
```diff
@@ -5,7 +5,7 @@ import { Link } from 'react-router-dom'
 const TeacherHeader = () => {
   return (
     <header className='teacher-header'>
-      <Link to='/teacherDashboard'>
+      <Link to='/'>
       <h1 className='header-title'>gumberoo</h1>
```
Comment (Melizzo) [inline src/TeacherHeader/TeacherHeader.js:9]: Ahh!! that makes sense!! I should have updated that in my PR.

File: src/App/App.js [hunk 6]
```diff
@@ -15,19 +15,19 @@ import { getLesson } from '../thunks/getLesson'
 import { setStudent, setLesson } from '../actions/index'
 import { bindActionCreators } from 'redux'
 
-function App(props) {
+function App() {
 
   return (
     <main className="App">
+        <Route exact path='/'>
```
Comment (KyleWong2510) [inline src/App/App.js:22]: hopefully this fixes it

**Review by Melizzo — APPROVED** [review]

File: src/StudentDashboard/StudentDashboard.js [hunk 7]
```diff
@@ -22,8 +24,12 @@ function StudentDashboard(props) {
   }
   const getTeachersLesson = async () => {
     await props.getLesson(props.lessonId)
-  } 
+  }
+  
+  const motivationalTalk = ['Good luck!', 'Great job!', 'You got this!', 'I want some bamboo', 'Nice pick!']
```
Comment (KyleWong2510) [inline src/StudentDashboard/StudentDashboard.js:29]: LOL i want some bamboo.  Great work with the animations

**Review by KyleWong2510 — APPROVED** [review]
Super stoked for you on those animations.  Killing it as always!

**Pull request merged**